Week05/lesson10/classwork/city_changer_starter_code/js/index.js

```diff
@@ -0,0 +1,53 @@
+// User types in a city name
+// User clicks on Update button to submit
+// Word is matched with an image
+// Background image changes accordingly
+
+
+// Use the below to prevent the default behavior of the browser which in this case is to refresh the current page.
+//
+// function switchCity(event){
+//   event.preventDefault();
+// }
+//
+
+
+function switchCity(event) {
+
+  event.preventDefault();
+
+  var city = document.querySelector("#city-type").value;
```

Comment:
in jQuery: `var city = $('#city-type').val();`

Also, I noticed that after you type the city into the input, you don't clear the input, which would be nice UX for this assignment since there's on "clear" button. To do that, all you have to do is store the city, and once you have it you can use jQuery's `val()` function to clear it, like so:

```
var city = $('#city-type').val();
$('#city-type').val('');
city = city.toLowerCase().trim();
```

...also remember that there's a lot of power in *reassigning* variables and you can use the existing variable value while reassigning, that's what I'm doing above. Because otherwise, it'd be forced to make a new variable every time I want to modify the value, just like you did (you have `city`, `citylow` and `citylowtrim`). Not there's anything wrong with how you did it, but it's just cleaner to have one variable instead of three